rest-api/core-rest/src/test/java/uk/gov/gchq/gaffer/rest/service/v2/OperationServiceV2IT.java

```diff
@@ -356,7 +356,7 @@ public OperationFieldPojo() {
         }
 
         public OperationFieldPojo(final String name, final String className, final boolean required,
-                final String summary, final Set<String> options) {
+                                  final String summary, final Set<String> options) {
```

Comment:
What's this change doing here? I think this was introduced in a merge by mistake.